packages/trace-viewer/src/ui/aiConversation.tsx

```diff
@@ -66,10 +93,10 @@ export function AIConversation({ conversationId }: { conversationId: string }) {
           placeholder='Ask a question...'
           className='message-input'
         />
-        {conversation.isSending() ? (
+        {onCancel ? (
```

Comment:
I prefer still doing an explicit check, rather than swapping out a callback. It's weird from an API perspective to have APIs that literally become `undefined` if their conditions aren't met.